Jenkinsfile

```diff
@@ -181,174 +183,140 @@ spec:
+                                                        }
+                                                        """]
+                                                    )
+                                                }
 
-                                            writeFile([file: 'kubeprod-manifest.jsonnet', text: """
-(import "manifests/platforms/gke.jsonnet") {
-  config:: import "${clusterName}-autogen.json",
-  letsencrypt_environment: "staging",
-  prometheus+: import "tests/testdata/prometheus-crashloop-alerts.jsonnet",
-}
-"""
-                                            ])
-                                        }{
-                                            // update glue records in parent zone
-                                            container('gcloud') {
+                                                writeFile([file: "${env.WORKSPACE}/${clusterName}/kubeprod-manifest.jsonnet", text: """
+                                                    (import "${env.WORKSPACE}/src/github.com/bitnami/kube-prod-runtime/manifests/platforms/gke.jsonnet") {
+                                                        config:: import "${env.WORKSPACE}/${clusterName}/${clusterName}-autogen.json",
+                                                        letsencrypt_environment: "staging",
+                                                        prometheus+: import "${env.WORKSPACE}/src/github.com/bitnami/kube-prod-runtime/tests/testdata/prometheus-crashloop-alerts.jsonnet",
+                                                    }
+                                                    """]
+                                                )
+                                            }{
+                                                def nameServers = []
                                                 withEnv(["PATH+JQ=${tool 'jq'}"]) {
-                                                    def nameServers = []
-                                                    def output = sh(returnStdout: true, script: "gcloud dns managed-zones describe \$(gcloud dns managed-zones list --filter dnsName:${dnsZone} --format='value(name)' --project ${project}) --project ${project} --format=json | jq -r .nameServers")
-                                                    for (ns in readJSON(text: output)) {
-                                                        nameServers << ns
+                                                    container('gcloud') {
+                                                        def output = sh(returnStdout: true, script: "gcloud dns managed-zones describe \$(gcloud dns managed-zones list --filter dnsName:${dnsZone} --format='value(name)' --project ${project}) --project ${project} --format=json | jq -r .nameServers")
+                                                        for (ns in readJSON(text: output)) {
```

Comment:
This explicit "copy" loops can go away if/when we change insertGlueRecords to accept java.util.List fwiw (you may as well do it in this PR).  (here and in AKS)

Reply:
done! PTAL